data/json/vehicleparts/tanks.json

```diff
@@ -142,6 +142,37 @@
     "damage_reduction": { "all": 28, "stab": 12 },
     "variants": [ { "symbols": "O", "symbols_broken": "#" } ]
   },
+  {
+    "id": "fridgetank_large",
+    "type": "vehicle_part",
+    "name": { "str": "refrigerated tank (100L)" },
+    "item": "fridgetank_large",
+    "location": "fuel_source",
+    "broken_color": "red",
+    "durability": 170,
+    "description": "A 100L refrigerated tank.  When turned on, it will cool the liquid inside, extending the time until the food spoils.",
+    "categories": [ "cargo" ],
+    "damage_modifier": 80,
+    "epower": "-80 W",
+    "breaks_into": [
+      { "item": "steel_lump", "count": [ 2, 2 ] },
+      { "item": "steel_chunk", "count": [ 1, 2 ] },
+      { "item": "scrap", "count": [ 1, 5 ] }
```

Comment:
Yet again, item made of aluminum but breaks into steel. Don't swap materials like that.